manifest.json

```diff
@@ -22,15 +22,17 @@
       "scripts/background.js"
     ]
   },
-  "browser_action": {
+  "page_action": {
+    "show_matches": ["*://*/*"],
+    "hide_matches": ["*://www.instapaper.com/*"],
```

Comment:
The Open Reading List action is still useful on Instapaper.com. It’s difficult to get there from Isntapaper’s mobile web page as they don’t link to the front page from an open article. I don’t think this is necessary.

Reply:
I see. Well, in order to provide a more consistent behavior with the actual addon, I should replace
```
    "show_matches": ["*://*/*"],
    "hide_matches": ["*://www.instapaper.com/*"],
```
with:
```
    "show_matches": ["*://*/*"],
    "hide_matches": [
      "*://www.instapaper.com/u*",
      "*://www.instapaper.com/read/*",
      "*://www.instapaper.com/api/*",
      "*://www.instapaper.com/text?u=*"
    ],
```
which closely follows https://github.com/da2x/instapaper-webextension/blob/87df338a7af6ae303c17a1e37ea5ec9ed721b755/scripts/const.js#L24-L28

Reply:
You can hide it for everything but `www.instapaper.com/read/*`, I think. That is the only page I know where it’s actually useful.